### PR TITLE
Aran output transform

### PR DESCRIPTION
**Pull request by aranXmos**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,9 +18,9 @@ list(APPEND BUILD_FLAGS_Darwin "-Wall" )
 list(APPEND BUILD_FLAGS_Darwin "-fsanitize=undefined" )
 list(APPEND BUILD_FLAGS_Darwin "-fsanitize=integer" )
 list(APPEND BUILD_FLAGS_Darwin "-fsanitize=implicit-conversion" )
-list(APPEND BUILD_FLAGS_Darwin "-fsanitize=address" )
```
Comment (andrewstanfordjason) [inline CMakeLists.txt:21]: why change this?

File: README.md [hunk 2]
```diff
@@ -20,12 +20,14 @@ This document assumes familiarity with the XMOS xCORE architecture, the XMOS too
 
 ## Building
 For an XCore build:
+
     mkdir -p build_xcore
     cd build_xcore
     cmake -DCMAKE_TOOLCHAIN_FILE=../etc/xmos_toolchain.cmake ..
     make
```
Comment (andrewstanfordjason) [inline README.md:27]: can these changes be removed?

**Review by andrewstanfordjason — CHANGES_REQUESTED** [review]
would you mind applying llvm formatting to the pr?

**Comment by andrewstanfordjason** [comment]
I've applied the formatting

File: README.md (hunk 2 above)
Comment (aranXmos) [reply README.md:27]: The newline makes it render as a code block instead of a paragraph

**Review by andrewstanfordjason — APPROVED** [review]

**Pull request merged**